### DISCUSSION TITLE
Auto change brightness

### DISCUSSION THREAD

**Discussion by cric1** [seerge/g-helper#399] [Q&A]
When I plugged in my laptop with Armoury Crate installed, the brightness level was automatically set to 100. when disconnecting, the brightness setting returned to the level it was previously set to. However, the brightness level does not adjust automatically with this. Is there a way to auto change brightness? 
Thanks

**Comment by seerge** [comment]
@cric1 i think in your case it happened cause ac was using different power plans for plugged / battery states. And it is windows that saves and changes brightness with power plans. 

"This" (G-helper) works with current power plan, and therefore brightness doesn't change. If you really want brightness to change - you can create couple of plans and set them via config as explained here https://github.com/seerge/g-helper#custom-power-plans-with-each-mode

**Reply by cric1** [reply]
Alright thank you very much
